components/automate-chef-io/content/docs/backup.md

```diff
@@ -52,7 +52,7 @@ Some users prefer to store backups offline in single-file archives. Such archive
 include both the configuration data and the reporting data contained in the standard
 backup.
 
-The [configured]({{< ref "backup.md#backup-to-filesystem" >}}) backup directory contains both the timestamp-based directory for the configuration, and the reporting data stored in the `automate-elasticsearch-data` directory.
+The [configured]({{< ref "backup.md#backup-to-a-filesystem" >}}) backup directory contains both the timestamp-based directory for the configuration, and the reporting data stored in the `automate-elasticsearch-data` directory.
```

Comment:
```suggestion
The [configured backup directory]({{< ref "backup.md#backup-to-a-filesystem" >}}) contains both the timestamp-based directory for the configuration, and the reporting data stored in the `automate-elasticsearch-data` directory.
```

idk if thats right, but something a bit more descriptive for the link text would be rad

Reply:
fixed